packages/repositories.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
+  <repository path="../src/NUnitEngine/Addins/nunit.v2.driver.tests/packages.config" />
   <repository path="../src/NUnitFramework/testdata/packages.config" />
   <repository path="../src/NUnitFramework/tests/packages.config" />
   <repository path="..\src\NUnitEngine\Addins\nunit.v2.driver.tests\packages.config" />
```

Comment:
Seems never to have been a problem. However, I'm not sure why the repositories.config file is changing at all as a part of this, so I'll check. Also, the added line is actually the same path as the line three below it!